Gemfile

```diff
@@ -11,3 +11,5 @@ end
 gem 'asciidoctor', '~> 1.5.4'
 gem 'coderay', '1.1.1'
 
+
+gem "webrick", "~> 1.7"
```

Comment:
What was this for? I pulled it down, installed webrick, and things looked ok locally, so I assume this is fine.

Reply:
Not sure, when I tried to run it locally, I had to update webrick for it to work. I didn't realize that it changed the gem file.

Reply:
Let me merge this and watch. If something goes wrong I'll back it out.